cli/Cargo.toml

```diff
@@ -45,10 +45,10 @@ winapi = "=0.3.9"
 winres = "=0.1.11"
 
 [dependencies]
-deno_ast = { version = "0.9.0", features = ["bundler", "codegen", "dep_graph", "module_specifier", "proposal", "react", "sourcemap", "transforms", "transpiling", "typescript", "view", "visit"] }
+deno_ast = { version = "0.10.0", features = ["bundler", "codegen", "dep_graph", "module_specifier", "proposal", "react", "sourcemap", "transforms", "transpiling", "typescript", "view", "visit"] }
 deno_core = { version = "0.117.0", path = "../core" }
-deno_doc = "0.26.0"
-deno_graph = "0.18.0"
+deno_doc = "0.28.0"
+deno_graph = "0.21.1"
 deno_lint = { version = "0.22.0", features = ["docs"] }
```

Comment:
By the way, deno_lint and dprint-plugin-typescript were both released with deno_ast 0.10.0.